pkg/ebpf/events_processor.go

```diff
@@ -232,18 +234,10 @@ func (t *Tracee) processSchedProcessExec(event *trace.Event) error {
 				if ok && hashInfoObj.LastCtime == castedSourceFileCtime {
 					currentHash = hashInfoObj.Hash
 				} else {
-
-					// ring1
-					err = capabilities.GetInstance().Required(func() error {
-						currentHash, err = computeFileHashAtPath(sourceFilePath)
-						if err == nil {
-							hashInfoObj = fileExecInfo{castedSourceFileCtime, currentHash}
-							t.fileHashes.Add(capturedFileID, hashInfoObj)
-						}
-						return nil
-					})
-					if err != nil {
-						logger.Errorw("Requiring capabilities", "error", err)
+					currentHash, err = computeFileHashAtPath(sourceFilePath)
```

Comment:
Let's also add a comment here about required capability already in base ring (for the sake of the future us)